packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -320,17 +320,26 @@ export class GrpcPlugin extends BasePlugin<grpc> {
     const plugin = this;
     return (original: typeof grpcTypes.makeGenericClientConstructor): never => {
       plugin._logger.debug('patching client');
-      return function makeClientConstructor<ImplementationType>(
+      return function makeClientConstructor(
         this: typeof grpcTypes.Client,
-        methods: grpcTypes.ServiceDefinition<ImplementationType>,
+        methods: { [key: string]: { originalName?: string } },
```

Comment:
Is this `ServiceDefintion` type removed for better readability?

Reply:
Yes, that's right.